oaknational/Oak-Web-Application
# Fix/mathjax provider just for mathjax lessons LESQ-774

## Description

Music year: {owa_music_year}

- Limits lesson overview page mathjax usage - no legacy subject and only from allowed list if they contain mathjax in lesson data.


Allowed subject slugs:

  "maths",
  "physics",
  "chemistry",
  "biology",
  "combined-science",
  "science",

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2365--oak-web-application.netlify.thenational.academy
Check allowed lessons with MathJax
Type window.MathJax into the console and press Enter. If MathJax is loaded, you should see an object returned that represents the MathJax library. If it returns undefined, then MathJax is not loaded on the page. You may need to clear the cache between tests


## Checklist
